03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,49 @@
-<!DOCTYPE html>
-<html>
-    <head>
-        <title>The Daily Planet</title>
-        <link rel="stylesheet" href="css/daily-planet-style.css"/>
-    </head>
-    <body>
-        <!-- You code start here -->
-    </body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>The Daily Planet</title>
+    <link rel="stylesheet" href="css/daily-planet-style.css" />
+</head>
+
+<body>
+    <header>
+        <h1>Daily Planet</h1>
+        <nav>
+            <a href="http://espn.go.com/">Sports</a>
+            <a href="http://kxan.com">Local</a>
+            <a href="http://www.wsj.com">Economy</a>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <article>
+                <video width="320" height="240" autoplay loop>
+                  <source src="https://i.imgur.com/wFovN4O.mp4" type="video/mp4">
+                </video>
+            </article>
+            <article>
+                <h2>Bill Clinton Loves Balloons!</h2>
+                <p>Last night at the Democratic National Convention, we learned a number of things. Chelsea Clinton kind of sounds like Mia Thermopolis from The Princess Diaries if you close your eyes and squint your ears. "Believing in science" is somehow not a given for a presidential candidate in 2016. And most importantly, Bill Clinton, possible future First Gentleman of the United States, loves balloons.</p>
+                <br>
+                <br>
+                <p>Bill Clinton loves balloons truly, madly, deeply. He loves them like they are the morning sun and the evening moon. He loves them the way most people love finding a $50 bill in their winter coat. The internet, a collection of humans with the skill to purify crude wells of joy into barrels of pure energizing happiness, did excellent work with this revelation.</p>
+            </article>
+        </section>
+        <aside>
+            <img src="http://static4.businessinsider.com/image/56f1953652bcd028008b8102-1000-1415/tollthis.jpg" alt="'Toll this' Burger King ad">
+        </aside>
+    </main>
+    <footer>
+        <a href="mailto:christopherperkins.512@gmail.com">Email me!</a>
+        <br>
+        <p>Phone: 555-555-5555</p>
+        <div class="footright">
+            <a href="https://www.facebook.com/DailyPlanetRVA/"><img src="https://cdn0.iconfinder.com/data/icons/yooicons_set01_socialbookmarks/256/social_facebook_box_blue.png" alt="Facebook"></a>
+            <a href="http://www.twitter.com"><img src="http://icons.iconarchive.com/icons/limav/flat-gradient-social/512/Twitter-icon.png" alt="Twitter"></a>
+            <a href="http://www.instagram.com"><img src="http://www.democrathotsprings.com/Instagram-icon-small.png" alt="Instagram"></a>
+        </div>
+    </footer>
```

Comment:
You are missing just one requirement for project: Add copyright and company name to footer. If you can add this in and recommit, I can bump your grade up to 100.